django_server/.github/workflows/deploy.yml

```
@@ -0,0 +1,218 @@
+            docker tag anythingllm:previous anythingllm:current
+            docker-compose up -d
+          EOF
+
+      - name: Notify Slack - Rollback
+        uses: slackapi/slack-github-action@v1.24.0
+        with:
+          channel-id: 'deployments'
+          slack-message: |
+            ⚠️ Production Rollback Executed
+            Failed Commit: ${{ github.sha }}
+            Rolled back to previous version
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)